samples/struct.js

```diff
@@ -193,7 +193,8 @@ async function queryWithArrayofStruct(instanceId, databaseId, projectId) {
     sql:
       'SELECT SingerId FROM Singers ' +
       'WHERE STRUCT<FirstName STRING, LastName STRING>(FirstName, LastName) ' +
-      'IN UNNEST(@names)',
+      'IN UNNEST(@names) ' +
+      'ORDER BY SingerId',
```

Comment:
Seems like this change is unrelated to what the PR description is aiming to fix. Was this snuck in on accident?

Reply:
Well, not completely on accident. Strangely enough, the system test using this example failed repeatedly for this PR (see the build log of the above commits). I have not been able to find any logical reason why the change to `SessionLeakError` should have any impact on this test, but I made the change to get the build successful.
I do agree however that it would probably be better to move that change into a separate PR. I'll look into that.

If you do have any logical explanation for why the change to `SessionLeakError` could have any (indirect) impact on this test, then I'm all ears. :-)
Or is this test known to be flaky?

Reply:
Looks like it is failing in our latest release PR, not really sure what changed? 🤷‍♂